src/Persistence/Array_.php

```diff
@@ -14,21 +14,23 @@
  */
 class Array_ extends Persistence
 {
-    /**
-     * Array of data.
-     *
-     * @var array
-     */
-    public $data;
+    /** @var array */
+    private $data;
```

Comment:
Maybe better `protected` because what if I extend this class in my project? Then I still would like to access this property from inside that class.

Reply:
answered on Skype, can be changed later if absolutely needed, but we should target more on apis than access to a raw underlaying data...